az/components/deploy-ecs-proxies.yml

```diff
@@ -0,0 +1,41 @@
+steps:
+  - bash: |
+      set -e
+      
+      dist_dir="$(realpath $(Pipeline.Workspace)/$(SERVICE_NAME)/$(SERVICE_ARTIFACT_NAME))"
+      proxy_vars_file=""
+      
+      if [ -f  ${dist_dir}/ecs-deploy-$(APIGEE_ENVIRONMENT).yml ]; then
+          proxy_vars_file="${dist_dir}/ecs-deploy-$(APIGEE_ENVIRONMENT).yml"
+      else
+          if [ -f  ${dist_dir}/ecs-deploy-all.yml ]; then
+            proxy_vars_file="${dist_dir}/ecs-deploy-all.yml"
+          fi
+      fi
+      
+      
+      if [ ! -z "${proxy_vars_file}" ]; then
+      
+      # get primary aws config
+      mkdir -p ~/.aws
+      mv $(Agent.TempDirectory)/ptl_aws_config ~/.aws/config
+      chmod 400 ~/.aws/config
+      
+      # ensure deployment pre-reqs
+      source $(Pipeline.Workspace)/$(SERVICE_NAME)/$(SERVICE_ARTIFACT_NAME)/.build_env_vars
+      
+      account=ptl SERVICE_BASE_PATH=$(SERVICE_BASE_PATH) APIGEE_ENVIRONMENT=$(APIGEE_ENVIRONMENT) \
```

Comment:
Can you parameterise anything that refers to `ptl`? We'll also have `prod` in the near future.